lib/safe_commit.rb

```diff
@@ -51,28 +59,51 @@ def test_files(options = { safety: true })
   end
 
   def branch
-    `git branch --show-current`
+    `git branch --show-current`.chomp
   end
 
   def trunk_branch(setting)
     ModiFile.instance.protected_branch = setting
   end
 
   def be_trunk
-    "#{ModiFile.instance.protected_branch}\n"
+    ModiFile.instance.protected_branch.to_s
   end
 
   # linting DSL
 
   def linting
```

Comment:
We are _theoretically_ allowing some flexibility in testing engines. Should we follow a similar approach for linting? I.e. wrap the logic in here inside a case statement.

Reply:
yes it should defer to the source project.